components/staticmenu/items.htm

```diff
@@ -1,15 +1,17 @@
-{% for item in items if not item.viewBag.isHidden %}
-    <li role="presentation" class="{{ item.isActive ? 'active' : '' }} {{ item.isChildActive ? 'child-active' : '' }} {{ item.viewBag.cssClass }}">
-        {% if item.url %}
-            <a href="{{ item.url }}" {{ item.viewBag.isExternal ? 'target="_blank"' }}>
-                {{ item.title }}
-            </a>
-        {% else %}
-            <span>{{ item.title }}</span>
-        {% endif %}
+{% for item in items %}
+    {% if item.viewBag['isHidden'] is not defined or not item.viewBag.isHidden %}
+        <li role="presentation" class="{{ item.isActive ? 'active' : '' }} {{ item.isChildActive ? 'child-active' : '' }} {{ (item.viewBag['cssClass'] is defined) ? item.viewBag.cssClass }}">
+            {% if item.url %}
+                <a href="{{ item.url }}" {{ (item.viewBag['isExternal'] is defined) ? 'target="_blank"' }}>
```

Comment:
```suggestion
                <a href="{{ item.url }}" {{ (item.viewBag['isExternal'] is defined and item.viewBag['isExternal'] == true) ? 'target="_blank"' }}>
```

Reply:
I'm not sure I want to encourage this, Twig isn't exactly the best place to be doing variable existence checks, so it might be better if we just made sure those properties always existed instead.